packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec

```
@@ -4,25 +4,21 @@
 
 %global gem_name pulp_2to3_migration_client
 
-%global release 1
-%global prerelease rc1
-%global prereleaserpm %{?prerelease:.}%{?prerelease}
-
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.1
-Epoch: 1
-Release: %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}
-Summary: Pulp 2 to Pulp 3 API Ruby Gem
+Version: 0.2.0b1
```

```suggestion
Version: 0.2.0
```
and then set `prerelease b1` above